### PR TITLE
Lesson 4 classes

### PR DESCRIPTION
**Pull request by diogoaurelio**
Adds solutions for exercises lesson 2 and enables showing lesson 4 about classes

### CODE REVIEW

File: oop-1-class-constructor/index.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 ---
-title: OOP 1 What is a class
-nav_order: 0
-has_children: true
-nav_exclude: true
+title: "Lesson 4: OOP: What is a class"
```
Comment (olayinkasf) [inline oop-1-class-constructor/index.md:2]: Let's change this to just `4 - ....` to match this format in the ToC

image

Comment (olayinkasf) [reply oop-1-class-constructor/index.md:2]: I think `OOP - Objects, Classes & Constructor` is more descriptive.

File: oop-1-class-constructor/index.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
+nav_order: 4
+has_children: false
+nav_exclude: false
 ---
 
-# Lesson 7: Object-Oriented Programming (OOP) 1
+# Lesson 4: Object-Oriented Programming (OOP)
```
Comment (olayinkasf) [inline oop-1-class-constructor/index.md:8]: I think `OOP - Objects, Classes & Constructor` is more descriptive.

File: recap-loops/homework-solutions.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 title: "Assignment Solutions"
 nav_order: 2
 parent: "2 - Recap loops"
```
Comment (olayinkasf) [inline recap-loops/homework-solutions.md:4]: This parent should be `3 - Data Structure - Arrays and Lists`

File: oop-1-class-constructor/index.md (hunk 2 above)
Comment (olayinkasf) [inline oop-1-class-constructor/index.md:8]: I think `Lesson 4: OOP - Objects, Classes & Constructor` is more descriptive.

File: oop-1-class-constructor/index.md [hunk 4]
```diff
@@ -19,7 +19,7 @@ nav_exclude: true
 
 ## Recap & Assignment check
 
-Let's look into the assignment from lesson 6
+Let's look into the assignment from lesson 3
 
 [Google Slides](https://docs.google.com/presentation/d/1M_Eeyv9fA4oJda54TCXdyhBqFoclJgqYenxT7GqkXwo/embed)
```
Comment (NancyMSoliman) [inline oop-1-class-constructor/index.md:25]: are we intending to add slides or it's by mistake? we didn't use slides in previous sessions

Comment (olayinkasf) [reply oop-1-class-constructor/index.md:25]: The slides are publicly available to be used. Feel free to use them if you wish. I recommend that you do for this particular session.

**Pull request merged**